neo/Network/P2P/TaskManager.cs

```diff
@@ -224,8 +224,8 @@ private void RequestTasks(TaskSession session)
             }
             if ((!HasHeaderTask || globalTasks[HeaderTaskHash] < MaxConncurrentTasks)
                 && (Blockchain.Singleton.HeaderHeight < session.StartHeight
-                    || (Blockchain.Singleton.HeaderHeight == session.StartHeight
-                        && Blockchain.Singleton.HeaderHeight == sessions.Select(x => x.Value.StartHeight).Max()
+                    || (Blockchain.Singleton.Height == Blockchain.Singleton.HeaderHeight
+                        && Blockchain.Singleton.HeaderHeight >= sessions.Select(x => x.Value.StartHeight).Max()
                         && TimeProvider.Current.UtcNow.ToTimestamp() - 60 >= Blockchain.Singleton.GetBlock(Blockchain.Singleton.CurrentHeaderHash)?.Timestamp)))
```

Comment:
it should probably be `TimeProvider.Current.UtcNow.ToTimestamp() - 60000` here for master3.x, because we changed to `ms`. Could you double check @yongjiema?

In addition, please set this wait time in a variable before using, it will become more clear.

Reply:
@vncoelho Seems like it's the same. https://github.com/neo-project/neo/blob/master/neo/Helper.cs#L170

Reply:
Hey @yongjiema, this function is the same, correct, but the general use of timestamp changed. 
I will add a PR with these comments.

Could you please change to `ToTimestampMS`. Otherwise it will not be consistent with `Blockchain.Singleton.GetBlock(Blockchain.Singleton.CurrentHeaderHash)?.Timestamp`.

In addition, fell free to further investigate. We may have missed something during this timestamp changes.

Reply:
Maybe we should rename `Block.Timestamp` to `TimestampMS`.

Reply:
I agree, @erikzhang, can we proceed with that refactoring in another PR?